server/src/domain/sql_backend_handler.rs

```diff
@@ -3,7 +3,7 @@ use async_trait::async_trait;
 
 use lldap_domain_handlers::handler::{BackendHandler, RequestContext, WithContextHandler};
 
-#[derive(Clone)]
+#[derive(Clone, Debug)]
```

Comment:
Erm, what are you hoping to get from this debug implementation?

I'd rather have a custom debug impl that doesn't print anything, rather than a bunch of implementation details. Then there's the question of _why_ you need it to be Debug

Reply:
It's just a leftover from some debugging, I won't need it, I'm pretty sure, and I'll also go over all of these additions to existing types to make sure they are actually needed in the final result.

Reply:
Removed, including some others.